runtime/vam/expr/function/network_of.go

```diff
@@ -0,0 +1,213 @@
+package function
```

Comment:
Nit: We don't use underscores in Go file names (except for test files and build constraints).